src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +238,36 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display Settings help banner on Event Ticket's Settings.
+	 *
+	 * @since TBD
+	 */
+	public function maybe_display_ticket_settings_help_banner() {
+
+		// Bail if we aren't in Events > Settings.
+		if ( 'tribe-common' !== tribe_get_request_var( 'page' ) ) {
+			return;
+		}
+
+		// Bail if not at wp-admin > Events > Settings > Tickets tab.
+		if ( 'event-tickets' !== tribe_get_request_var( 'tab' ) ) {
+			return;
+		}
+
+		/** @var \Tribe\Tickets\Admin\Settings\Service_Provider $settings */
+		$settings = tribe( 'tickets.admin.settings' );
+
+		echo $settings->render_settings_banner();
+
+//		tribe_notice(
```

Comment:
Using the `tribe_notice` method will require us to change the template a lot and adjust CSS again to make it compatible.

Reply:
With the relocation using this method is looking broken on load: https://www.loom.com/share/a20208ce334c45f793e7e0ee3b09a75c

If you check it's re-positioned and it's not a great experience.